docs/jwst/stpipe/user_step.rst

```diff
@@ -13,27 +13,30 @@ parameters on it.
 Steps can be configured by either:
 
     - Writing a configuration file
-
     - Instantiating the Step directly from Python
 
+.. _running_a_step_from_a_configuration_file:
+
 Running a Step from a configuration file
 ========================================
 
-A Step configuration file is in the well-known ini-file format.
-stpipe uses the `ConfigObj
-<https://configobj.readthedocs.io/en/latest/>`_ library to parse
-them.
+A Step configuration file contains one or more of a ``Step``'s parameters. Any
+parameter not specified in the file will take its value from the CRDS-retrieved
+configuration or the defaults coded directly into the ``Step``. Note that any
+parameter specified on the command line overrides all other values.
 
-Every step configuration file must contain the ``name`` and ``class``
-of the step, followed by parameters that are specific to the step
-being run.
+The preferred format of configuration files is the :ref:`config_asdf_files`
+format. Refer to the :ref:`minimal example<asdf_minimal_file>` for a complete
+description of the contents. The rest of this documented will focus on the step
+parameters themselves.
 
-``name`` defines the name of the step.  This is distinct from the
-class of the step, since the same class of Step may be configured in
-different ways, and it is useful to be able to have a way of
-distinguishing between them.  For example, when Steps are combined
-into :ref:`stpipe-user-pipelines`, a Pipeline may use the same Step class
-multiple times, each with different configuration parameters.
+All step parameters appear under the ``parameters:`` section and indented. The
+amount of indentation does not matter, as long as all parameters are indented
+equally.
+
+Every step configuration file must contain the parameter ``class``, followed by
```

Comment:
The example in lines 28 - 30 of config_cfg.rst shows `name` first, then `class`.  Does the order matter?

Reply:
No, order does not matter.